components/src/core/components/Divider/Divider.vue

```diff
@@ -18,15 +23,28 @@ export default defineComponent({
     style: {
       type: Object,
     },
+    orientation: {
+      type: String,
+      default: AREA_ORIENTATION_HORIZONTAL,
+      validator: function(value: AreaOrientation) {
+        return ORIENTATION_TYPES.indexOf(value) !== -1;
+      }
+    }
   },
 
   computed: {
     classes(): object {
+      if (this.orientation == 'vertical') {
+        return { 'oxd-divider-vertical': true };
```

Comment:
can't we write this as 

return this.orientation == 'vertical' ? { 'oxd-divider-vertical': true } : { 'oxd-divider-horizontal': true };

Reply:
Yes thilanka that's better. I have changed that.